bloom-housing/bloom
# fix: public multiple photos from detroit

# Pull Request Template

## Issue Overview

This PR addresses #3345


## Description

Issue pulls multiple images from detroit, both in partners (upload images), and public (display in listing view)
UIC changes  required included here: https://github.com/bloom-housing/ui-components/pull/96.

## How Can This Be Tested/Reviewed?

use 3345/Public-Multiple-Photos branches both for UIC and Bloom.
Few listings will have populated multiple photos after reseed -> check file changes
Go to partners check if photos can be added (and if that updates public)
test at least 4 variants -> 1, 2, 3, 4 and more photos
check if primary photo can be changed.
check if photos can be removed.

There were changes to modal (in UIC), if you can think of any modal that could be broken after that, let me know.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
